core-bundle/src/Routing/Content/ArticleResolver.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\Content;
+
+use Contao\ArticleModel;
+use Contao\PageModel;
+
+class ArticleResolver implements ContentUrlResolverInterface
+{
+    public function resolve(object $content): ContentUrlResult
+    {
+        if (!$content instanceof ArticleModel) {
+            return ContentUrlResult::abstain();
+        }
+
+        return ContentUrlResult::resolve(PageModel::findWithDetails($content->pid));
```

Comment:
To test this, it probably needs the ContaoFramework with the adapter.

Reply:
Fixed in ba10d117a0f61274cad33d47158e880347e5efb1